### PR TITLE
Fix: Stub default config file in specs

### LINKED ISSUE #149: Missing mock in tests, /etc/SUSEConnect is read from the current system
If `/etc/SUSEConnect` does not exist in the system the tests run correctly.

If you create `/etc/SUSEConnect` file with some configuration, for example:

``` yaml

---
url: https://foo.suse.com
insecure: false
```

Then the tests fail:

```
  1) SUSE::Connect::Client.new empty_opts should set url to default_url
     Failure/Error: subject.url.should eq subject.class::DEFAULT_URL

       expected: "https://scc.suse.com"
            got: "https://foo.suse.com"
```

The config file from system was read instead of using the default.

### PR DESCRIPTION
**Pull request by ngetahun**
This PR attempts to fix #149 issue. 

Currently tracked by https://trello.com/c/PRLriO3z/1795-project-maintenance-connect-rmt-yast-rmt

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/378527/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 253ef511a0b7ce5c32bb6fbfacfac6557c4ed4eb into 79cf9110cc9556e54af264c66277b56d3aeec151)

File: spec/connect/client_spec.rb [hunk 1]
```diff
@@ -18,6 +21,7 @@
 
   subject { client_instance }
 
+
```
Comment (wstephenson) [inline spec/connect/client_spec.rb:24]: extra whitespace?

**Review by wstephenson — APPROVED** [review]
lgtm functionally

File: spec/connect/client_spec.rb (hunk 1 above)
Comment (ngetahun) [reply spec/connect/client_spec.rb:24]: Rubocop added that :shrug:

**Pull request merged**